templates/style/rustdoc-common.scss

```diff
@@ -98,6 +98,15 @@ div.rustdoc {
                     top: $top-navbar-height;
                 }
             }
+
+            // A later version of rustdoc uses the .sidebar.shown instead of .sidebar.mobile, and
+            // also needs a height adjustment.
+            &.shown {
+                width: 250px;
+                margin-left: 0;
+                top: 45px + $top-navbar-height !important;
```

Comment:
I think you need a `calc()` here.

Reply:
Also, why the `45px`?

Reply:
45px is the size of the rustdoc topbar.

`calc()` is not needed because this is uses [sass simplification](https://sass-lang.com/documentation/values/calculations#simplification). Basically sass does the math at compile time and emits `top: 78px` into the file.

Reply:
Oh that's cool. I didn't know that. :)